packaging/installer/kickstart-static64.sh

```diff
@@ -208,8 +208,6 @@ safe_sha256sum() {
 
 mark_install_type() {
     install_type_file="/opt/netdata/etc/netdata/.install-type"
-    # shellcheck disable=SC1090
-    . "${install_type_file}"
```

Comment:
`[ -e "${install_type_file}" ] && . "${install_type_file}"` would be the best fix here!